Mars_Application_UserStory2_TestAutomation/Features/ExtendedPostive.feature

```diff
@@ -0,0 +1,50 @@
+﻿Feature: ExtendedPostive
+Test MARS application Test cases
+
+Background: 
+## Positive_TC _001 - Log into Mars Application
```

Comment:
can you please break your feature files per feature not per type of test! feature file should be named Langauge.feature, skill.feature etc and within those you will need to put the types of tests extended postive extended negative etc